Cargo.toml

```diff
@@ -12,7 +12,7 @@ license = "MIT OR Apache-2.0"
 keywords = ["IANA", "time"]
 categories = ["date-and-time", "internationalization", "os"]
 readme = "README.md"
-edition = "2018"
+edition = "2021"
```

Comment:
Please back out this change. There's a repo-wide PR for this up here: https://github.com/strawlab/iana-time-zone/pull/161.

If it is necessary to get this PR buildable, please update the base branch of this PR to that of #161.

Reply:
As documented, this change is required to support the new compatibility code generated by newer `windows-bindgen`. We can't back it out, we can however rebase this PR on top of yours as done now.